Makefile

```diff
@@ -4,13 +4,19 @@
 #
 # Authors:
 #   Jan Kiszka <jan.kiszka@siemens.com>
+#   Li Huaqian <huaqian.li@siemens.com>
 
 CROSS_COMPILE = arm-linux-gnueabihf-
 CC = gcc
 
 RTI_MODULE = 1
 
+ifndef PON_REASON_BASE_ADDR
 CFLAGS = -mcpu=cortex-r5 -DRTI_MODULE=$(RTI_MODULE)
+else
+CFLAGS = -mcpu=cortex-r5 -DRTI_MODULE=$(RTI_MODULE) \
+		 -DPON_REASON_BASE_ADDR=$(PON_REASON_BASE_ADDR)
+endif
```

Comment:
```
ifdef PON_REASON_BASE_ADDR
CFLAGS += -DPON_REASON_BASE_ADDR=$(PON_REASON_BASE_ADDR)
endif
```

Reply:
But If nothing speaks against using the IOT2050 default value of `PON_REASON_BASE_ADDR` on the TI EVM boards as well, let's keep that as default.

Reply:
Please help review again.